created Meeting minutes (#13)
DragonAxe/CSE423-Compiler-Collective: meeting_minutes/1-13-20.md

# Meeting Minutes: [01-13-2020]
## People Present [Remote or in person]:
[comment]: <> (*Please note who was present.*)
* Reece:    
    - [X] Present  
    - [ ] Absent
* Riley:    
    - [X] Present  
    - [ ] Absent
* Niko:    
    - [X] Present  
    - [ ] Absent
* Jeriah:    
    - [X] Present  
    - [ ] Absent

## Tasks discussed or worked on:
[comment]: <> (Brief descripiton of talking topics, ideas for further exploration, and questions to follow up on)

### Topics
* Decided on a langauge for the main compiler Project. Congrats Python.
* Set up main form of communication for team through Riot.
* Created Git Repo, decided on commit style and started creating files to check push and merge credentials.

### Research and Development

> * Look into Yap and other python formatters for cleaner code styles and for error checking.

Reviewer: ```suggestion
* Look into YAPF and other python formatters for automatic code styling
* Suggested utilization of Pyre for type-checking
* Suggested utilization of CircleCI or similar for continuous testing/integration
* Suggested utilization of PyLint for code linting
```

### Questions
* None as of yet.

## Future Road map and issues:
[comment]: <> (Please Link issues as they are created)

* Set up CI #1 : 1/17/2020 : [Unknown]

> * Set up Pyer #2 : 1/17/2020 : [Unknown]

Reviewer: ```suggestion
* Set up Pyre #2 : 1/17/2020 : [Reece]
```

* Set up YAPF #3 : 1/17/2020 : [Riley]
* Set up meeting minutes #6 : 1/17/2020 : [Jeriah]
* Set up Pylint #7 : 1/17/2020 : [Reece]

> * Set up Pylint #11 : 1/17/2020 : [unknown]

Reviewer: ```suggestion
* Set up Pytest #11 : 1/17/2020 : [unknown]
```

Author: Updated and made the requested changes.
